docs/user-wallets/pkps/overview.md

```diff
@@ -0,0 +1,48 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Overview
+
+Programmable Key Pairs (PKPs) are ECDSA public/private key pairs created by the Lit network using Distributed Key Generation (DKG). Each Lit node holds a share of the private key, and more than two-thirds of these shares must be collected to execute a given action (i.e. signing a transaction).
+
+PKPs can be programmed with custom logic to automate on-chain actions and implement sophisticated access control. They can be used to create non-custodial wallets, automate transactions, and build complex decentralized applications.
+
+Please refer to the [Quick Start Guide](./quick-start.md) for how to get started with PKPs.
+
+## How it Works
+
+### Key Generation and Storage
+
+PKPs are created using Distributed Key Generation (DKG) across the Lit network. This process ensures that the private key is never assembled in one place, and is instead split into shares with each node in the Lit network holding a share. Consequently, no single entity, including Lit Protocol itself, has access to the full private key. The public key, however, is publicly available and can be used to derive the corresponding blockchain address.
+
+### Signing Process
+
+When a signing operation is requested, a Lit node client sends the request to the Lit network where nodes holding key shares participate in a threshold signing process. Each Lit node is then tasked with providing their signature share of the data using their respective PKP key share. Once more than two-thirds of the Lit nodes respond, the signature shares are combined resulting in a complete signature of the data you requested be signed by the PKP.
+
+This threshold signing process ensures that the full private key is never reconstructed at any point, maintaining security while producing a signature with the PKP.
+
+### Programmability and Access Control
+
+A key feature of PKPs is their programmability. Each PKP can be authorized to execute specific [Lit Actions](../../sdk/serverless-signing/overview.md), which are JavaScript scripts that define the conditions and logic for key usage. This programmability allows for things like access control, where the use of a PKP can be gated by various on-chain conditions like token ownership, or off-chain conditions such as OAuth verification. Additionally, because a Lit Action is just a JavaScript script, you have a lot of flexibility on what your Lit Action does and how it makes use of the PKP.
```

Comment:
> This programmability allows for things like access control, where the use of a PKP can be gated by various on-chain conditions like token ownership

IMO, this relates more to our access control / encryption offering than PKPs, wouldn't you say? 

I think the functionality to emphasize here should be more about transaction automation, conditional signatures, verifying off-chain data, or defining sophisticated permissions for key usage (which is what i think you were trying to get at with the mention of access control)

Reply:
Refactored to:

A key feature of PKPs is their programmability through [Lit Actions](../../sdk/serverless-signing/overview.md), which are JavaScript scripts that define the conditions and logic for key usage. This programmability enables several advanced capabilities such as:

- Transaction Automation: PKPs can be programmed to automatically sign and execute transactions based on various conditions, streamlining complex on-chain operations.
- Conditional Signatures: Implement custom logic for when and how a PKP can sign transactions, based on various on-chain and off-chain factors.
- Off-chain Data Attestation: Use PKPs to cryptographically sign and attest to off-chain data, allowing for the secure integration of real-world information into blockchain systems.
- Sophisticated Permission Systems: Define granular and context aware permissions for key usage, allowing for fine-tuned control over who can use the PKP, and under what circumstances.

The flexibility of Lit Actions, being JavaScript-based, allows you to create complex and tailored solutions for managing and utilizing PKPs for you application.